### DISCUSSION TITLE
resizing picture heights in gallery of causes

### DISCUSSION THREAD

**Discussion by smsalmanshahAN** [an-nadaaa/nda-front#128] [Feedback]
In the slider gallery of the causes page.... the image is being cropped off or it is only dispaying the top left part of the picture.

Even on zooming in or out or changing to mobile view the same portion of picture is showing so it is responsively scaling the picture.

2 immediate solution comes to mind 
1. Resize the pictures to the height of the gallery section
2. Change the height of the gallery section

![image](https://user-images.githubusercontent.com/101787508/161238318-852c529b-c13d-4c8c-a283-2b2ce5a319b6.png)

**Comment by an-nadaa** [comment]
@smsalmanshahAN This is fixed
